packages/fields/package.json

```diff
@@ -9,7 +9,6 @@
 		"gutenberg",
 		"dataviews"
 	],
-	"private": true,
```

Comment:
I presume this needs to be rebased from `trunk`, correct? This package was made public already.